docs/source/download.rst

```diff
@@ -68,25 +98,30 @@ do:
 
    pip install --user git+https://github.com/devitocodes/devito.git
 
+Devito is also available as a `pip package`_ in PyPI.
+To install Devito (without the tutorials and examples), follow:
+
+.. code-block:: shell
 
-to install. Note that you don't need to get the code
-using git first in this method. 
+   pip install devito
```

Comment:
Add descriptions of addditional options, `pip install devito[extras,mpi,nvidia]`

Reply:
sure